manager.go

```diff
@@ -387,6 +387,40 @@ func (m *Manager) moduleChanged(
 	return changed, fmt.Sprintf("module %q changed because %s", mod.Source, why), nil
 }
 
+func (m *Manager) AddWantedOf(stacks []stack.S) ([]stack.S, error) {
+	wantedBy := map[string]stack.S{}
+	wanted := []stack.S{}
+
+	for _, s := range stacks {
+		wantedBy[s.PrjAbsPath()] = s
+		wanted = append(wanted, s)
+	}
+
+	visited := map[string]struct{}{}
+
+	for len(wantedBy) > 0 {
+		for _, s := range wantedBy {
+			ws, err := m.stackLoader.LoadAll(m.root, s.AbsPath(), s.Wants()...)
+			if err != nil {
+				return nil, err
+			}
+
+			for _, wws := range ws {
```

Comment:
```suggestion
			for _, wantedStack := range wantedStacks {
```

I know you like short names, but the ws/wws thing seems to have gotten too far IMHO :rofl:. But feel free to ignore.